meetings/2023-11-09.md

```diff
@@ -0,0 +1,29 @@
+# ISIS TC Meeting November 09, 2023 @ 1PM Arizona Time
+
+## [Teams Link](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2f_%23%2fl%2fmeetup-join%2f19%3ameeting_YWRkZjdiMGUtZWJlOC00OWMzLThlMTItZTk0Y2MyM2E1MWE0%40thread.v2%2f0%3fcontext%3d%257b%2522Tid%2522%253a%25220693b5ba-4b18-4d7b-9341-f32f400a5494%2522%252c%2522Oid%2522%253a%2522c27c6e98-e45a-45ff-aea5-7f10d6fe67c1%2522%257d%26anon%3dtrue&type=meetup-join&deeplinkId=e54b3969-3c7f-4efb-9cad-ee99cf639f86&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true)
+
+### Attending
+- Stuart Robbins 
+- Kris Becker 
+- Trent Hare 
+- Kelvin Rodriguez
+- 
+
+## Agenda / Notes
+- Any potential follow up to PSMRTS discussion (Kris)
+  - Last 3 objectives on discussion post are the most important 
+  - Gave a presentation yesterday, might be able to post links on discussion board 
+- Update on notifying users about support meetings (Kelvin)
+  - Implementing a newsletter subscription option that will be advertized by the stale bot
+
+## Discussions for next meeting
+- We can talk in person about the PR size RFC (Kelvin)
+- Should USGS make IAA progress public (Kelvin)
+- 
```

Comment:
I don't know why this is the purview of the ISIS TC. The IAA is a contractual mechanism used to move monies between government entities. This group helps with the governance of the ISIS project and has absolutely no authority over a (functionally opaque) agreement between two other entities.

If there is something that the ISIS TC is considering as it relates to CHANGELOG entries or discussions of forward planning as they relate to the project, that would be a very appropriate agenda item.

Reply:
It was brought up during last meeting and added last second, we can [remove it from the agenda still](https://github.com/planetarysoftware/ISIS_TC/pull/195)